src/main/java/lotto/domain/LottoNumberBox.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberBox {
+    private static final int LOTTO_MAX_NUMBER = 45;
+    private static final int LOTTO_MIN_NUMBER = 1;
+    private static final int ONE = 1;
+    private static final List<LottoNumber> LOTTO_NUMBERS = IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER + ONE)
```

Comment:
```suggestion
    private static final List<LottoNumber> LOTTO_NUMBERS;
    
    static {
        LOTTO_NUMBERS = IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER + ONE)
                .mapToObj(LottoNumber::new)
                .collect(Collectors.toList());
    }
```
static변수의 초기화 값에 로직이 필요하다면 static 블록을 사용해보는건 어떨까요? 
https://uoonleen.tistory.com/6

Reply:
감사합니다 수정했습니다!